package.json

```diff
@@ -64,7 +64,7 @@
     "loglevel": "^1.7.0",
     "make-fetch-happen": "^8.0.14",
     "minipass-fetch": "^1.3.3",
-    "qs": "~6.9.6",
+    "qs": "^6.10.1",
```

Comment:
@wjhsf do you think it was a `~` for compatibility reasons?

Reply:
My guess is that it is `~` because @bredmond-sf had npm config [save-prefix](https://github.com/SalesforceCommerceCloud/commerce-sdk/pull/57/files#diff-0b810c38f3c138a3d5e44854edefd5eb966617ca84e62f06511f60acc40546c7) set to `~` instead of `^`.

https://github.com/SalesforceCommerceCloud/commerce-sdk/pull/57/files#diff-0b810c38f3c138a3d5e44854edefd5eb966617ca84e62f06511f60acc40546c7